src/api/users.coffee

```diff
@@ -22,10 +24,19 @@ exports.authenticate = (email) ->
 
     if not user
       utils.logAndSetResponse this, 404, "Could not find user by email #{email}", 'info'
+      # User NOT authenticated, send audit
+      audit = atna.userLoginAudit 8, 'openhim', os.hostname(), email
```

Comment:
magic numbers: it'd be good to define the outcome codes somewhere - perhaps even in your atna audit lib. (handy [ref](https://github.com/jembi/uct-redcap/blob/master/acl-portal/backend/ATNA.coffee#L8) to just copy/paste from)